Explorer/Assets/Scripts/SceneRunner/Scene/ISceneFacade.cs

```diff
@@ -9,6 +9,8 @@ namespace SceneRunner.Scene
     public interface ISceneFacade : IUniTaskAsyncDisposable, IDisposable
     {
         SceneShortInfo Info { get; }
+        ISceneStateProvider? SceneStateProvider { get; }
+        SceneEcsExecutor? EcsExecutor { get; }
```

Comment:
The same here: it's not a nullable structure - it will always exist

Reply:
Handled here: https://github.com/decentraland/unity-explorer/pull/632/commits/94368455d9941fdb15ea2fa87119aa78eee8c34c